INSTALL.md

```diff
@@ -0,0 +1,71 @@
+### 1. Back-end setup
+
+Follow the [Sheer installation instructions](https://github.com/cfpb/sheer#installation)
+to get Sheer installed.
+**NOTE:** We suggest creating a virtualenv variable specific to this project,
+such as `cfgov-refresh` instead of `sheer` used in the Sheer installation instructions:
+
+```bash
+$ mkvirtualenv cfgov-refresh
+```
+
+Install the following dependencies into your virtual environment.
+We called ours `cfgov-refresh` (see previous step above):
+
+```bash
+$ workon cfgov-refresh
+
+$ pip install git+git://github.com/dpford/flask-govdelivery
+$ pip install git+git://github.com/rosskarchner/govdelivery
+```
+
```

Comment:
I think setting up the constants should be included here, even if it's just setting `WORDPRESS` in the command line